docs/backend-api/getting-started/authentication.md

```diff
@@ -1,29 +1,259 @@
 
-To interact with the Navixy API, you need an API key hash, which is essential for authenticating your requests. Follow these steps to obtain the hash.
+With Navixy APIs, you can use either the user session hash or API keys.
 
-## Steps to Obtain an API Key Hash
+An API Key, also known as a "Hash" is a randomly generated string used to verify and authenticate actions. The hash of the API key must be included in most API calls.
+
+### API Keys Advantages
+
+Unlike a user's session hash, the API key has the following characteristics:
+
+- It will not be deleted if the user logs out or changes their password.
+- There is no need to [renew](./user/session/index.md#renew) the key periodically.
+- It does not require the transfer or storage of the username and password.
+- It can be deleted at any time if there is a suspicion of compromise.
+- You can create a separate key for each individual integration.
+
+Every user account may have up to 20 API keys. Sub-users don't have API keys functionality.
+
+Follow these ways to obtain the API key.
+
+## Ways to Obtain an API Key Hash
+
+### Creating an API Key from User Interface
 
 1. **Login to the Web Interface**:
-    - Access the Navixy user interface via the web
-    - Use your credentials to log in
+    - Access the Navixy user interface via the web.
+    - Use your credentials to log in.
 
 2. **Navigate to the API Key Section**:
-    - Once logged in, go to the `Settings` menu.
-    - Find the `API Keys` section, typically located under `Account Settings` or `Security`.
+    - Once logged in, click on your username.
+    - Find and select the `API Keys` section.
 
 3. **Generate a New API Key**:
-    - Click on the `Generate New API Key` button.
+    - Click on the plus button on the top left corner.
     - Provide a name for the API key to easily identify it later.
-    - Select the permissions and scope for the key based on your needs.
-    - Click `Generate`.
+    - Click `Save`.
 
 4. **Copy the API Key Hash**:
-    - Once the key is generated, you will see the API key hash.
-    - Copy this hash and store it securely. You will use this hash in your API requests.
+    - Once the key is generated, you will see it in the API keys table, including the label, creation date, and the API key hash.
+    - Use this hash in your API requests.
 
-## Using the API Key Hash
+### Creating an API Key Using the User Session Hash
```

Comment:
Why create an API key via the API? It’s designed to be done through the UI - copy the key and paste it into your scripts. I don't think we should explain how to obtain a regular user session. Moreover, all of this is already described on the page https://developers.navixy.com/backend-api/resources/commons/api-keys/

Reply:
+1

Reply:
It is better to describe all possible ways since they exist and do it in one place. Searching for information piece by piece is not an excellent practice for documentation.

Reply:
Creating an API key through the API is not what a guide is needed for. Because this is not the recommended path. Why describe what should not be done?

Reply:
You're right. I will delete this information from my commit then

Reply:
Updated